cis/com.b2international.snowowl.snomed.cis.product/assembly/common/bin/cis.sh

```diff
@@ -80,6 +73,6 @@ exec $JAVA_EXECUTABLE $SO_JAVA_OPTS \
   -Djava.io.tmpdir="$TMP_DIR" \
   -Dosgi.install.area="$KERNEL_HOME" \
   -Dosgi.configuration.area="$CONFIG_AREA" \
-  -jar plugins/org.eclipse.equinox.launcher_1.6.300.v20210813-1054.jar -console 2501
+  -jar plugins/org.eclipse.equinox.launcher_1.6.600.v20231106-1826.jar -console 2501
```

Comment:
If `-Dosgi.console` works, it can be removed from this line (`cis.bat` is already updated this way):

```suggestion
  -jar plugins/org.eclipse.equinox.launcher_1.6.600.v20231106-1826.jar
```